changelog.md

```diff
@@ -1,3 +1,12 @@
+## Next version
+
+### Bugfixes
```

Comment:
Please re-use the same headings as used in the last version. Just add them all here.